doc/howtos/ip-address-config.rst

```diff
@@ -0,0 +1,103 @@
+.. _ip-address-config:
+
+Configuring an IP Address in the Ostro |trade| OS
+######################################################
+
+The Ostro Project uses Connection Manager (ConnMan), a Linux daemon and command-line utility, for managing internet connections 
+within embedded devices. 
+
+ConnMan supports these network technologies:
+
+* Ethernet, using a built-in card, connected via host or client side USB
+* WiFi, using wpa_supplicant
+* Celluar connectivity using oFono
+* Bluetooth connectivity using Bluez
+
+By default all technologies are disabled at the very first startup
+to prevent unwanted wireless or wired communication from happening. ConnMan remembers the state of network
+technologies over reboots. 
+(You can read more about ConnMan at https://01.org/connman/documentation.)
+
+
+Enabling Network Technologies 
+=============================
+
+Before a network transport (such as WiFi) can be used, it must be enabled in ConnMan. You can check to see what technologies
+have already been enabled with the command :: 
+
+   $ connmanctl technologies
+
+The command will indicate if LAN and WiFi technologies are enabled. If not, you can enable either (or both) of them with the commands:: 
+
+   $ connmanctl enable ethernet 
+   $ connmanctl enable wifi 
+
+
+Configuring a Static IP Address
+================================
+
+ConnMan refers to network devices as services. To configure a static IP address for a given LAN or WiFi service, 
+you first find the name of the service assigned by ConnMan and then provide the IP address information, as 
+shown in these steps: 
+
+#. List the services available on your device ::
+
+     $ connmanctl services
+
+     *AO Wired                ethernet_0008a209b525_cable
+
+   The service name discovered is ``ethernet_0008a209b525_cable``.
+
+   Note: The symbols in the output above are: '\*' favorite (saved) network, 'A' autoconnectable, 'O' online and 'R' ready. 
+   If no letter is shown in the O/R column, the network is not connected. In addition, temporary states include 
+   'a' for association, 'c' configuration and 'd' disconnecting. When any of these three letters are showing, 
+   the network is in the process of connecting or disconnecting. A network is in state 'ready' once it has 
+   obtained an IPv4 or IPv6 address or both, and can indicate that the network might still need a proxy set up to get 
+   internet connectivity.
+
+#. Once you know the service name, you can assign a static IP with the command::
+
+     $ connmanctl config <service> –ipv4 manual <ip address> <netmask> <gateway> 
+
+   For example, using the service name discovered above: ::
+
+     $ connmanctl config ethernet_0008a209b525_cable –ipv4 manual 192.168.1.4 255.255.255.0 192.168.1.1
+
+You can assign a static IP to a discovered WiFi adapter service in the same manner.
+
+
+Configuring a Dynamic (DHCP) IP Address
```

Comment:
I think it'd be more appropriate to call this section something like "Connecting to a WiFi network".